Modules/_zoneinfo.c

```diff
@@ -58,16 +58,16 @@ typedef struct {
     uint8_t month;
     uint8_t week;
     uint8_t day;
-    int8_t hour;
+    int16_t hour;
```

Comment:
Would you mind to write a short comment explaining that hour is in range [-167; 167) (RFC ...)? Since it's non obvious why minute and second use int8_t, but hour use int16_t.

Reply:
167 is written in many places, it would help to avoid inconsistencies if it's updated tomorrow.

I suggested to add a comment (ex: mentioning this constant) to explain why hour requires int16_t and not int8_t, where the structure is defined.